projects/mysadna2/src/app/mock-users.ts

```diff
@@ -0,0 +1,9 @@
+import {User} from './user';
+import {Project} from './project';
+import {Contribution} from './contribution';
+
+export const USERS: User[] = [
+    {userId: 1, userName: "litalm", firstName: "Lital", lastName: "Morgenstein", email: "lital.morg@gmail.com", socialNetworks: ["LinkedIn", "Github", "Twitter", "Facebook"], skills: ["Java", "swift", "dart", "C++", "bash", "Front-End development", "Mobile development", "Flutter", "Android", "iOS", "Android Studio", "Xcode", "Linux", "mac"], projects: [1, 2], userContributions: ["Added profile pages and project pages.", "Added profile and project pages.", "Added profile pages and project pages."]},
+    {userId: 2, userName: "litalm2", firstName: "Lital2", lastName: "Morgenstein2", email: "lital.morg2@gmail.com", socialNetworks: ["LinkedIn", "Github"], skills: ["Java", "C++"], projects: [1, 2, 3], userContributions: []},
```

Comment:
Some of files have lint errors. Did you see it?
To see errors: `ng lint`
To fix errors: `ng lint --fix`

Reply:
If your line exceed maximum line length and you can avoid it. Then you can say tslint to ignore it.
Put the comment before the line:
```
// tslint:disable-next-line
```

Reply:
Great job!